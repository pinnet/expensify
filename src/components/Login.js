import React from 'react';
import {connect} from 'react-redux';
import { startLogin } from './../actions/auth';

export const Login = ({startLogin}) => (
    <div>
        <p><button onClick={startLogin}>Login via Google</button></p>
    </div>
);
const mapDispatchToProps = (dispatch) => ({
    startLogin: () => dispatch(startLogin())
});

export default connect(undefined,mapDispatchToProps)(Login);