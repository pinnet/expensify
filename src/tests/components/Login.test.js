import React from 'react';
import Login from './../../components/Login'
import { shallow } from 'enzyme'; 

it('should render correctly', () => {
    const wrapper = shallow(<Login />);
    expect(wrapper).toMatchSnapshot();   
});
