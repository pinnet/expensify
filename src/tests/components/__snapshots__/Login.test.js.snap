// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <p>
    <button
      onClick={[Function]}
    >
      Login via Google
    </button>
  </p>
</div>
`;
